modules/jupyter-widget/src/create-deck.js

```diff
@@ -8,6 +8,13 @@ import {registerLoaders} from '@loaders.gl/core';
 
 import * as deck from './deck-bundle';
 
+// See https://github.com/uber/deck.gl/blob/d74e375ddef5e2ac222641a2b93e1f4287d668f1/examples/playground/src/configuration.js#L14
+import {DracoWorkerLoader} from '@loaders.gl/draco';
+import {Tiles3DLoader, CesiumIonLoader} from '@loaders.gl/3d-tiles';
+
+// Note: deck already registers JSONLoader...
+registerLoaders([CSVLoader, DracoWorkerLoader]);
```

Comment:
@ibgreen I thought we no longer have to register the DracoLoader for 3d-tiles to work?

Reply:
Yes that was the plan but sadly I didn't get that done in time for 2.1.